install.sh

```diff
@@ -2,21 +2,35 @@ set -e
 
 OS=''
 case `uname` in
-  Darwin*)  OS="macos" ;; 
+  Darwin*)  OS="macos" ;;
   Linux*)   OS="linux" ;;
   *)        echo "unknown os: $OSTYPE" && exit 1 ;;
 esac
 
 ARCH=`uname -m`
 case "$ARCH" in
-  ix86*|x86_64*)    ARCH="x86_64" ;; 
+  ix86*|x86_64*)    ARCH="x86_64" ;;
   arm64*|aarch64*)  ARCH="aarch64" ;;
   *)                echo "unknown arch: $ARCH" && exit 1 ;;
 esac
 
 
 export TAG="v1.0.0-rc4"
+export BINARYEN_TAG="version_116"
+
+
 curl -L -O "https://github.com/extism/js-pdk/releases/download/$TAG/extism-js-$ARCH-$OS-$TAG.gz"
+curl -L -O "https://github.com/WebAssembly/binaryen/releases/download/$BINARYEN_TAG/binaryen-$BINARYEN_TAG-$ARCH-$OS.tar.gz"
+
 gunzip extism-js*.gz
 sudo mv extism-js-* /usr/local/bin/extism-js
 chmod +x /usr/local/bin/extism-js
+
+tar xvf "binaryen-$BINARYEN_TAG-$ARCH-$OS.tar.gz"
+mv "binaryen-$BINARYEN_TAG"/ binaryen/
+sudo mkdir /usr/local/binaryen
+sudo mv binaryen/bin/wasm* /usr/local/binaryen
+
+for file in $(ls /usr/local/binaryen); do
+  sudo ln -s  /usr/local/binaryen/$file /usr/local/bin/$file
+done
```

Comment:
Could we try this instead? Do you mind testing this? There are a few problems here:

1. binaryen expects arm64 for the release arch and we use aarch64
2. we should not do this if wasm-merge is already present because it overrides it and the ln will fail too
3. we only need to install wasm-merge so i figured i'd keep it scoped to that.


```suggestion
if ! which wasm-merge &> /dev/null
then
  echo "Installing wasm-merge..."

  case "$ARCH" in
    aarch64*)  ARCH="arm64" ;;
  esac

  curl -L -O "https://github.com/WebAssembly/binaryen/releases/download/$BINARYEN_TAG/binaryen-$BINARYEN_TAG-$ARCH-$OS.tar.gz"

  tar xvf "binaryen-$BINARYEN_TAG-$ARCH-$OS.tar.gz"
  mv "binaryen-$BINARYEN_TAG"/ binaryen/
  sudo mkdir /usr/local/binaryen
  sudo mv binaryen/bin/wasm-merge /usr/local/binaryen/wasm-merge
  sudo ln -s /usr/local/binaryen/wasm-merge /usr/local/bin/wasm-merge
else
  echo "wasm-merge already installed"
fi
```

Reply:
Ah these suggestions are good, If we only need `wasm-merg` then that is much simpler. I'll make changes shortly. Thank you.